tests/postprocessing_test.py

```diff
@@ -318,6 +319,14 @@ def test_steep_upslope(self):
             steep_upslope[-1]
             )
 
+    def test_consecutive_manoeuvres(self):
+        valid_manoeuvres, double_dips = consecutive_manoeuvres(
```

Comment:
Now you only test valid manoeuvres. It might be good to also test whether it returns false for invalid manoeuvres